scripts/release-tools/kube-burner-configs/cluster-density/endpoint-load-deployment.yml

```diff
@@ -0,0 +1,64 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: endpoint-load-{{.Replica}}
+spec:
+  replicas: {{.podReplicas}}
+  selector:
+    matchLabels:
+      app: endpoint-load-{{.Replica}}
+  template:
+    metadata:
+      labels:
+        app: endpoint-load-{{.Replica}}
+    spec:
+      imagePullSecrets:
+      - name: {{.JobName}}-{{.Replica}}
+      containers:
+      - image: quay.io/rhacs-eng/qa:berserker-1.0-38-ge3b2927865
```

Comment:
This berserker image is not a main image.

Reply:
What do you mean by that?

Reply:
This image is from a PR not the main branch. That PR is now merged so I will fix this soon and use an image built from the main branch.

Reply:
Let us know when that is fixed, so we can review this PR.

Reply:
I have updated the berserker image tag.